src/main/scala/wdl4s/MemorySize.scala

```diff
@@ -0,0 +1,39 @@
+package wdl4s
+
+import wdl4s.parser.MemoryUnit
+
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+import scalaz.Scalaz._
+
+object MemorySize {
+  val memoryPattern = """(\d+(?:\.\d+)?)\s*(\w+)""".r
+  def parse(unparsed: String): Try[MemorySize] = {
```

Comment:
I'll see what I can do... the reason why I chose this approach is that I want the return type of `RuntimeAttributes.evaluate(...)` to be a `Map[String, Try[WdlValue]]`.  Since that function  is part of a public API, I wanted to avoid returning scalaz objects from it.  It seemed to make the most sense to to the conversion to `Try` here, though there are a few places I can do the conversion.